designs/2023-only-run-reporting-rules/README.md

```diff
@@ -48,46 +48,62 @@ should verify that the `rulePredicate` option is a function, and replace it with
 function if not. `processOptions` in `eslint-helpers.js` should also perform a validation check
 that the `rulePredicate` option is a function.
 
+The new `rulePredicate` function when implemented would look like this, using the `--quiet` flag
+rules outlined in this RFC as an example:
+
+```typescript
+linter.verifyAndFix(text, configs, {
+    rulePredicate: ({ ruleId: string, rule: Rule, severity: number }) => {
+        return severity === 2;
+    }
+});
+```
+
+This predicate would return true for all rules which have a severity of 2 (error).
+
 The alteration to the `--quiet` flag would be implemented by passing a predicate function
```

Comment:
I think we are still missing the details around this. How does the rule predicate function end up passed to `Linter`? It would need to start out in `cli.js`, passed through to `ESLint` or `FlatESLint`, and only then end up at `Linter`.

Reply:
I believe I cover this two paragraphs down from here - it mentions each step it's passed down through

Reply:
Oops, sorry about that. It looks like I was viewing the "change since you last viewed" instead of the whole doc.